src/app/routes/public-forms.server.routes.js

```diff
@@ -271,12 +271,12 @@ module.exports = function (app) {
     forms.formById,
     publicForms.isFormPublicCheck,
     CaptchaMiddleware.checkCaptchaResponse,
-    encryptSubmissions.validateEncryptSubmission,
+    EncryptSubmissionMiddleware.validateAndProcessEncryptSubmission,
```

Comment:
wait, so `develop` is using both the javascript and typescript versions? I think it would be good to delete all the code that you're replacing so this doesn't happen!

Reply:
thanks for highlighting, i've removed it!